addons/mail/models/discuss/ir_config_parameter.py

```diff
@@ -0,0 +1,19 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import re
+
+from odoo import api, models
+from odoo.exceptions import ValidationError
+
+
+class IrConfigParameter(models.Model):
+    _inherit = 'ir.config_parameter'
+
+    @api.constrains("value")
+    def _validate_link_preview_regexp(self):
+        for record in self:
+            if record.key == "mail.mail_link_preview_ignore_list":
```

Comment:
You can save indentation (and possibly lines too) with either
```suggestion
        for record in self.filtered(lambda r: r.key == "mail.mail_link_preview_ignore_list"):
```
or (at the cost of lines)
```suggestion
        for record in self:
            if record.key != "mail.mail_link_preview_ignore_list":
                continue
```

Reply:
Vote for first one, (clearer)